CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ if(UNIX)
   find_package(GnuTLS REQUIRED)
   find_package(PkgConfig)
   find_package(Boost REQUIRED)
-  set(Boost_USE_STATIC_LIBS ON)
```

Comment:
We need this on for certain features to work.

Reply:
you can conditionally turn this on for RHEL if you want to continue with this PR. We do not want to turn it on for CentOS

Reply:
Thanks for your review, Nara. I understand this PR will be breaking changes for Ubuntu.
Could you please consider to update the installation steps on wiki for RHEL8?